front/app/containers/Admin/ideas/routes.tsx

```diff
@@ -35,14 +34,7 @@ export default () => ({
         </PageLoading>
       ),
     },
-    {
```

Comment:
I don't fully understand what is the `IdeaPreviewIndex` is and why it can be removed here? Is this the component where the idea statuses were still shown instead of the proposal statuses? Does this mean we will no longer have it in the input manager?

Reply:
I don't think this route is used at all. The idea preview is only used in the side modal for the input manager. So I deleted it.